WtWSMS/common/landed_titles/BP_e_britannia.txt

```diff
@@ -3188,60 +3960,25 @@ e_britannia = {
 			c_hereford = {
 				color = { 110 45 35 }
 				cultural_names = { #Added Welsh localization to this file. 
-					#Roman Cultures
-					name_list_roman = cn_ariconium
-					name_list_romano_gallic = cn_ariconium
-					name_list_romano_hispanic = cn_ariconium
-					name_list_romano_british = cn_ariconium
-					name_list_romano_african = cn_ariconium
-					name_list_romano_raetian = cn_ariconium
-					name_list_romano_thracian = cn_ariconium
-					name_list_romano_illyrian = cn_ariconium
-					name_list_romano_pannonian = cn_ariconium
-					name_list_dalmatian = cn_ariconium
-					name_list_romano_noric = cn_ariconium
-					name_list_romano_gothic = cn_ariconium
-					name_list_romano_dacian = cn_ariconium
-					name_list_romano_frank = cn_ariconium
-					name_list_romano_aramean = cn_ariconium
-					name_list_romano_vandalic = cn_ariconium
```

Comment:
Again here you remove content

Reply:
> Again here you remove content

Because Ariconium isn't Hereford. But since we already decided to re-add Segodunum as Bamburgh, I re-added Ariconium-Hereford as well.

Reply:
> > Again here you remove content
> 
> Because Ariconium isn't Hereford. But since we already decided to re-add Segodunum as Bamburgh, I re-added Ariconium-Hereford as well.

In my next commit, I will change it again. The closest Roman city to Hereford was Magnis and not Ariconium, which instead will replace Clifford.